ansible/roles/ccache/tasks/main.yaml

```diff
@@ -1,7 +0,0 @@
-- name: Install ccache
-  become: true
-  ansible.builtin.apt:
-    name:
-      - ccache
-    state: latest
-    update_cache: true
```

Comment:
I cannot find any mention of `ccache` within the autoware repository in this branch, why was it removed?

Reply:
Found it in the dockerfile below
https://github.com/autowarefoundation/autoware/pull/4072#discussion_r1504768094